### DISCUSSION TITLE
Add dropdown component inside a card component

### DISCUSSION THREAD

**Discussion by Carmen-Mercado** [saadeghi/daisyui#1176] [Q&A]
How can I add a dropdown component inside a card component?

**Comment by RyanTippsTX** [comment]
I think the issue is that DaisyUI dropdown is implemented with inline elements, and the border of the card is cutting off the overflow. 

Trythis:
1. add the class `flex` to the parent `<div>` that also has the dropdown class
2. set your `<ul>` component to be positioned absolute w / respect to its parent by adding these classes `absolute top-full right-0`

You might have to modify the [top / right / left / bottom ](https://tailwindcss.com/docs/top-right-bottom-left)as needed. You can also mess with the margins on your `<ul>` component too.

**Reply by RyanTippsTX** [reply]
@Carmen-Mercado let me know if this works for you

**Reply by Carmen-Mercado** [reply]
Thank you, I just added `overflow-visible` in the div with the class card, which resolved the bug.
